src/nn/context.ts

```diff
@@ -110,4 +110,27 @@ export class MLContext {
         tf.version_core}/dist/`);
     return tf;
   }
+
+  /** @internal */
+  // Expose wasm interface for supporting configure threads for wasm backend.
+  get wasm(): unknown {
+    return wasm;
+  }
+
+  /** @internal */
+  // Print which WASM binary is used by WASM backend.
+  async printWasmBinary(): Promise<void>  {
```

Comment:
If there is a tf.js API to get it, we should use it. Otherwise, we probably won't need this implementation, because this looks like a duplication of tf.js code. If tf.js changes their implementation, this implementation would be outdated.

Reply:
No, the `getPathToWasmBinary()` method isn't exported from `@tensorflow/tfjs-backend-wasm` module.

Reply:
Thanks for the confirmation. Then we probably won't do it by ourselves. Because this is mainly for debugging, one can either check the tf.Env() or the download wasm file in console.

WDYT?

Sorry for the confusion.

Reply:
OK, I will update commit to remove it.

Reply:
Done, please take another look, thanks.

Reply:
@huningxin gentle ping, PTAL updated PR, thanks.